### ISSUE TITLE
Add agents kind to REST query.

### ISSUE THREAD

**Issue by ZenMaster91** [Arquisoft/Agents_i3a#4]
For the REST web service that allows to query info about an agent the input must be changed from 
```json 
{"login": user, "password": password}
```
 to 
```json
{"login": user, "password": password, "kind":kind}
```
Notice that the `kind` field is new and its representation is an integer number that represents the agent's kind. The keyword identifier comes from a master CVS file that contains 2 fields separated by commas, where first field is the numeric code and the second one is the name of the user kind. For example:

```cvs
1,Person
2,Entity
3,Sensor
```
If the agents kind name changes in the master file the system must be able to resolve the new ones

**Comment by ZenMaster91** [comment]
Working on it

**Comment by ZenMaster91** [comment]
Implemented at #26
